cmd/bosun/conf/conf.go

```diff
@@ -69,21 +70,29 @@ type Conf struct {
 	squelch         []string
 }
 
-type AuthorizedIPs []*net.IPNet
-
-func (authorizedIPs AuthorizedIPs) Authorized(ip net.IP) (authorized bool) {
-	if len(authorizedIPs) == 0 {
+func (c *Conf) PutAuthorized(ip net.IP) bool {
+	// First process all blocked put ip ranges.
+	if len(c.BlockedPutIPs) > 0 {
```

Comment:
Because the default behavior is block which I want to keep in place in case this is ever extended. len of zero means (whitelist feature is not being used), so just allow if it isn't blocked